crates/typst/src/layout/align.rs

```diff
@@ -170,9 +171,9 @@ impl Alignment {
     #[func(title = "Inverse")]
     pub const fn inv(self) -> Alignment {
         match self {
-            Self::H(h) => Self::H(h.inv()),
-            Self::V(v) => Self::V(v.inv()),
-            Self::Both(h, v) => Self::Both(h.inv(), v.inv()),
+            Self::H(x) => Self::H(x.inv()),
```

Comment:
I think we should perhaps standardize the variable names as h, v for consistency with the H, V variants and generic type parameters elsewhere.

Reply:
Happy to! I was worrying renaming too many variables would ruffle too many feathers.